src/main/java/scrumweb/issue/controller/IssueController.java

```diff
@@ -34,4 +37,10 @@ public IssueDetailsDto createIssue(@PathVariable String projectKey, @RequestBody
     public IssueDetailsDto getDetails(@PathVariable Long id) {
         return issueService.getDetails(id);
     }
+
+    @GetMapping("/search/{paramkey}")
+    @ResponseStatus(HttpStatus.OK)
+    public List<IssueDto> findIssuesByKeyQuery(@PathVariable String paramkey){
+        return issueService.findIssuesByKeyQuery(paramkey);
```

Comment:
metoda nie potrzebna

Reply:
potrzebujesz tylko jedną metodę na BE która będzie przyjmowała query i zwracała SearchResultDto w którm będzie się znajdowały dwie listy IssueDto i ProjectDto